### ISSUE TITLE
Improved dark mode/updated color system

### ISSUE THREAD

**Issue by seeddisperser** [webb-tools/webb-dapp#584]
**Is your UI update related to a problem? Please describe.**
This update is responding to two problems
1. dark mode issues
2. overly complicated color system causing design drift. 

**Describe the solution you'd like (include image or link to prototype)**
To improve the dark mode, the monochromes have been updated, with a blue tint added to the darker colors. See the [dark mode implemented here](https://www.figma.com/file/35GtMH3QSbmmCpV7ddSZZu/%F0%9F%8C%89--Bridge-UI?node-id=573%3A49358).

At the same time, the color system has been implemented with clear roles for each color used. See the [updated color system here](https://www.figma.com/file/H43lYnTCOAPjO9kTrrepe1/%F0%9F%95%B8%EF%B8%8F-Design-System?node-id=30%3A342).

**Describe alternatives you've considered**
You can see other [dark mode explorations here](https://www.figma.com/file/H43lYnTCOAPjO9kTrrepe1/%F0%9F%95%B8%EF%B8%8F-Design-System?node-id=1198%3A11184). 

**Technical considerations**
New color values are as follows:
```
:root {

    /* Monochrome */
    --mono00: #fff; /* Card Background Light Mode */
    --mono20: #F7F8F7; /* Body Background Light Mode */
    --mono40: #E2E5EB; /* h1-h5 text Dark Mode */
    --mono60: #D3D8E2;
    --mono80: #C2C8D4;
    --mono100: #9CA0B0; /* P color Dark Mode */
    --mono120: #6C7180; /* P color Light Mode */
    --mono140: #4E5463;
    --mono160: #3A3E53;
    --mono170: #2E3244; /*new to improve dark mode*/
    --mono180: #2B2F40; /* card Background Dark Mode */
    --mono190: #252836; /*new to improve dark mode*/
    --mono200: #1F1D2B; /* h1-h5 Light Mode */

    /*light mode*/
    --purple10: #F6F4FF;
    --purple40: #B5A9F2;
    --purple70: #624FBE; /*primary*/
    --purple90: #4B3AA4;

    --blue10: #ECF4FF;
    --blue40: #9BC5FC;
    --blue70: #3D7BCE; /*primary*/
    --blue90: #23579D;

    --green10: #EBFFF0;
    --green40: #85DC8E;
    --green70: #288E32; /*primary*/
    --green90: #01550A;

    --yellow10: #FFFAEA;
    --yellow40: #FFE07C;
    --yellow70: #EAB612; /*primary*/
    --yellow90: #8D721A;

    --red10: #FFEDE4;
    --red40: #FFB18B;
    --red70: #EF570D; /*primary*/
    --red90: #A0370B; 
    
    
    /*dark mode*/
    --purple120: #221C41;
    --purple90: #6756BC;
    --purple50: #C6BBFA; /*primary*/
    --purple30: #E7E2FF;

    --blue120: #181F2B;
    --blue90: #23579D;
    --blue50: #81B3F6; /*primary*/
    --blue30: #B8D6FF;

    --green120: #0C270F;
    --green90: #01550A;
    --green50: #4CB457; /*primary*/
    --green30: #85DC8E;

    --yellow120: #3F3517;
    --yellow90: #8D721A;
    --yellow50: #F8D567; /*primary*/
    --yellow30: #FFEAA6;

    --red120: #422417;
    --red90: #A0370B;
    --red50: #FF874D; /*primary*/
    --red30: #FFCEB7; 

  }
```

**Additional context**
Because this editing process has removed some color variables, this update will require some minor style changes to be made to many of the elements in the design system.

**Comment by dutterbutter** [comment]
Related: #599

**Comment by dutterbutter** [comment]
Closed in #628
